apps/src/code-studio/pd/form_components/ButtonList.jsx

```diff
@@ -27,15 +29,20 @@ class ButtonList extends React.Component {
     answers: PropTypes.arrayOf(
       PropTypes.oneOfType([
         // Standard string answer
+        // See typedef SimpleAnswer in utils
         PropTypes.string,
 
         // or an answer followed by an input for additional text
+        // See typedef ExtraInputAnswer in utils
```

Comment:
nit: given the running sentence of the existing comments, could we put these new comments in parentheses or something similar?

Reply:
oooh, nice catch!